docs/data/charts/legend/legend.md

```diff
@@ -71,6 +71,16 @@ For the `pie` series, the `labelMarkType` property is available for each of the
 
 {{"demo": "LegendMarkType.js", "hideToolbar": true, "bg": "playground"}}
 
+#### Custom shapes
```

Comment:
Might be too edge case to mention, but the reason the default shapes are HTMLs with SVGs inside is to make gradients/patterns work out of the box. If you provide a purely HTML element they won't work.

Reply:
Yep https://deploy-preview-16739--material-ui-x.netlify.app/x/react-charts/styling/#gradients-and-patterns

Reply:
Added a warning 👍